src/Autocomplete.js

```diff
@@ -63,29 +62,32 @@ class Autocomplete extends Component {
       m,
       l,
       offset,
+      minLength,
       ...props
     } = this.props;
 
     const _id = 'autocomplete-input';
-    const _autocomplete = 'autocomplete';
     const _ul = 'autocomplete-content dropdown-content';
+    const sizes = { s, m, l };
+    let classes = {};
+    constants.SIZES.forEach(size => {
+      classes[size + sizes[size]] = sizes[size];
+    });
 
     return (
       <Row>
-        <Col s={s} m={m} l={l} offset={offset} className={cx('input-field', className)} {...props}>
+        <Col offset={offset} className={cx(cx('input-field', className), classes)} {...props}>
```

Comment:
Didn't think that cx took more than two arguments. I guess I should look at the specs, huh?